include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -3129,7 +3130,11 @@ namespace internal
    * Interpolates the values and gradients into the points specified in
    * @p compute_values_of_array() with help of the precomputed @p shapes.
    */
-  template <int dim, typename Number, typename Number2, int n_values = 1>
+  template <int dim,
+            typename Number,
+            typename Number2,
+            int  n_values    = 1,
+            bool no_renumber = false>
```

Comment:
```suggestion
            bool do_renumbering = true>
```

Reply:
How much influence has this template parameter?

Reply:
It is visible for `p=2` and `p=3`. especially for `dim=2`. (Up to 5% throughput)